tests/unit/test_request_absolute_endpoint.py

```diff
@@ -9,9 +9,7 @@ def test_request_absolute_endpoint():
     endpoint = "/api/service/health"
     full_url = f"http://localhost:9100{endpoint}"
     responses.add(responses.GET, full_url, json={})
-
     client = Client(UsernamePasswordAuth("username", "password"))
-
```

Comment:
Extraneous whitespace change; please remove.

Reply:
this newline is desired. please don't remove the newline. please remove this change.